static/locales/ur.yaml

```
@@ -0,0 +1,810 @@
+# Put the name of your locale in the same language
+Locale Name: 'اردو' 
+FreeTube: 'FreeTube'
+# Currently on Subscriptions, Playlists, and History
+'This part of the app is not ready yet. Come back later when progress has been made.': >-
+  'ایپ کا یہ حصہ ابھی تک تیار نہیں ہے۔ جب پیش رفت ہو جائے تو بعد میں واپس آئیں۔'
+
+# Webkit Menu Bar
+File: 'فائل'
+New Window: 'نئی کھڑکی'
+Quit: 'چھوڑو'
+Edit: 'ترمیم'
+Undo: 'کالعدم'
+Redo: 'دوبارہ کریں۔'
+Cut: 'کاٹنا'
+Copy: 'کاپی'
+Paste: 'چسپاں کریں۔'
+Delete: 'حذف کریں۔'
+Select all: 'تمام منتخب کریں'
+Reload: 'دوبارہ لوڈ کریں۔'
+Force Reload: 'زبردستی دوبارہ لوڈ کریں۔'
+Toggle Developer Tools: 'ڈویلپر ٹولز کو ٹوگل کریں۔'
+Actual size: 'اصل سائز'
+Zoom in: 'زوم ان کریں۔'
+Zoom out: 'دور کرنا'
+Toggle fullscreen: 'پوری اسکرین کو ٹوگل کریں۔'
+Window: 'کھڑکی'
+Minimize: 'کم سے کم کرنا'
+Close: 'بند کریں'
+Back: 'پیچھے'
+Forward: 'آگے'
+Open New Window: 'نئی ونڈو کھولیں۔'
+
+Version $ is now available!  Click for more details: 'ورژن $ اب دستیاب ہے! کلک کریں۔
+   مزید تفصیلات کے لیے'
+Download From Site: 'سائٹ سے ڈاؤن لوڈ کریں۔'
+A new blog is now available, $. Click to view more: 'ایک نیا بلاگ اب دستیاب ہے، $.
+   مزید دیکھنے کے لیے کلک کریں۔'
+Are you sure you want to open this link?: 'کیا آپ واقعی اس لنک کو کھولنا چاہتے ہیں؟'
+
+# Search Bar
+Search / Go to URL: 'تلاش کریں / یو آر ایل پر جائیں۔'
+Search Bar:
+  Clear Input: 'ان پٹ صاف کریں۔'
+  # In Filter Button
+Search Filters:
+  Search Filters: 'فلٹرز تلاش کریں۔'
+  Sort By:
+    Sort By: Sort By
```

I think the non translated ones should be set to empty strings
Ex:
```suggestion
    Sort By: ''
```